pediatric_rootlets/pediatric_rootlets.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# This script performs:
+# - segmentation of rootlets from T2w data (model model-spinal-rootlets_ventral_D106_r20240523),
+# - segmentation of spinal cord from T2w data (contrast agnostic model)
+# - segmentation of vertebral levels from T2w data (sct_label_vertebrae)
+# - detection of PMJ from T2w data (sct_detect_pmj).
+#
+# Authors: Katerina Krejci, Jan Valosek
+#
+
+# Uncomment for full verbose
+set -x
+
+# Immediately exit if error
+set -e -o pipefail
+
+# Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
+trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
+
+# Retrieve input params
+SUBJECT=$1    # sub-0001/ses-01
+
+# get starting time:
+start=`date +%s`
+
+# SCRIPT STARTS HERE
+# ==============================================================================
+# Display useful info for the log, such as SCT version, RAM and CPU cores available
+sct_check_dependencies -short
+
+# Go to folder where data will be copied and processed
+cd $PATH_DATA_PROCESSED
+
+# Copy source images
+cp -r $PATH_DATA/${SUBJECT%/*} .    # %/* - delete everything after last slash (/)
+
+# Go to anat folder where all structural data are located
+cd ${SUBJECT}/anat
+
+# Run model-spinal-rootlets_ventral_D106_r20240523 for dorsal and ventral rootlets segmentation
+# NOTE: as the model for both ventral and dorsal rootlets is not part of SCT yet, we run directly the nnUNet model using
+# the wrapper script run_inference_single_subject.py from the model-spinal-rootlets repository
+# https://github.com/ivadomed/model-spinal-rootlets/blob/main/packaging_ventral_rootlets/run_inference_single_subject.py
+# NOTE: we use SCT python because it has nnUNet installed 
+# NOTE: the command below expects that you downloaded the model (https://github.com/ivadomed/model-spinal-rootlets/releases/tag/r20240523) and saved it to:  ~/models/model-spinal-rootlets_ventral_D106_r20240523
+
+file_t2=${SUBJECT}_rec-composed_T2w.nii.gz
```

Comment:
@KaterinaKrejci231054 you could add a condition checking whether `rec-composed_T2w.nii.gz` exists, if not, you can use `acq-top_run`

Reply:
Resolved in [15de9f9](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/15de9f961bf51c2e8616f59b3ca25c6d8a42c0e4).